cmd/openshift-install/create.go

```diff
@@ -236,21 +236,25 @@ func destroyBootstrap(ctx context.Context, directory string) (err error) {
 	)
 	if err != nil {
 		logrus.Error(errors.Wrap(err, "waiting for bootstrap-complete"))
+		if err == wait.ErrWaitTimeout {
+			return err
```

Comment:
Why ignore the other kinds of errors?  I'd rather just revert #615 now that we're comfortable with the robustness of #606.

Reply:
sounds good to me.  I'll remove this from this PR, keep the log msg edits in.

Reply:
removed